docs/source/conf.py

```diff
@@ -441,13 +452,67 @@ def check_timeout_and_api_kwargs_presence(obj: object) -> int:
     )
 
 
+def create_return_admonitions() -> dict[str, str]:
+    """Creates 'Returned in' admonitions for classes that are returned in Bot's methods."""
+
+    methods_for_class = defaultdict(list)
+
+    # checking for islower() to avoid camelCase methods
+    for method in [m for m in dir(telegram.Bot) if not m.startswith("_") and m.islower()]:
```

Comment:
maybe `inspect.getmembers(telegram.Bot, inspect.is_method)` is a bit better? 🤔 Should avoid the try-except below …

Reply:
Done